bindings/pyroot_experimental/PyROOT/src/PyzPythonHelpers.cxx

```diff
@@ -19,12 +19,104 @@ PyROOT extension module.
 
 #include "CPyCppyy.h"
 #include "CPPInstance.h"
+#include "ProxyWrappers.h"
 #include "PyROOTPythonize.h"
 #include "RConfig.h"
 #include "TInterpreter.h"
 
 #include <sstream>
 
+////////////////////////////////////////////////////////////////////////////
```

Comment:
Following the logic of my previous comment, I wonder if we should put this in a file called `RVecPyz.cxx`, instead of just considering it a helper.

Reply:
I think it belongs in `RVecPyz.cxx`. Even if it does not pythonize the `RVec` class, it is related to it. Having a file for free functions could be an option, but I tend to dislike code files where you just put anything you don't know where to put.

Reply:
I think @etejedor has a point. For the maintainability moving it there it's an advantage, i.e. we know were to find it...